pynicotine/gtkgui/userbrowse.py

```diff
@@ -930,9 +930,9 @@ def on_download_directory_to_selected(self, selected, recurse):
     def on_download_directory_to(self, *args, recurse=False):
 
         if recurse:
-            str_title = _("Select Destination for Downloading Folder with Subfolders from User")
+            str_title = _("Select Destination Folder for this Download (with subfolders)")
```

Comment:
Perhaps "this" is not necessary here?

Reply:
That works somewhat in the English language, but makes translating the sentence a lot harder.

So, this is what happens to the version with this:
> Zielordner für diesen Download auswählen (mit Unterordnern) 

And this is the one without.
> Wählen Sie den Zielordner für den Download (mit Unterordnern)

So, the first sentence, because of the "this" the translator knows what belongs to what and produces a very clear translation.

With the second, the translator has to guess what belongs to what. It's still a correct sentence, but the sentence is already harder to read, since it does not start with the subject.

See also: https://www.dorisandbertie.com/goodcopybadcopy/2017/02/14/sentence-structure-how-to-make-your-sentences-much-easier-to-read

Reply:
In this case, it matters not that the sentance makes sense, rather what is more important here is the clarity of which direction the Folder is going. It is important that we must disambiguate between selecting a local Folder to Download _into_, as opposed to selecting a folder to be downloaded.

This was why I originally included "from".

Reply:
Your proposal looks good, but as is translates to:

> Wählen Sie das Ziel zum Herunterladen mehrerer Ordner vom Benutzer aus 

Assuming you don't speak German, that sentence misses the point completely.

But let's make two small changes:

> Select Destination folder for Downloading Multiple Folders from %(user)s

Now we specify the destination type, and don't add the word User, but do add the name of the user and presto:

> Wählen Sie den Zielordner zum Herunterladen mehrerer Ordner von %(user)s
> Selecteer de doelmap voor het downloaden van meerdere mappen van %(user)s 

That's what we wanted to say.

Reply:
I am concerned that putting obnoxious user names with special characters and whatnot in the dialog box title might cause unforeseen problems and be visually undesirable in many cases.

Reply:
OK, that statement contradicts what you previously proposed, but not to worry. The following sentence is very translatable.

> Select Destination folder for Downloading Multiple Folders

And I'm sure people will understand they are downloading from a user.

Reply:
I think we should avoid duplicating the word "folder" twice, because this creates ambiguity.

Why can we not have `Select Destination for Downloading Multiple Folders` ?

Reply:
Now that you mention it…

Reply:
Thank you for your input on this @hboetes your expertise with the language is very much appreciated :)

Reply:
Thanks. I was actually surprised how we could have this long discussion, but now that I think about it, it really makes sense, saying what you want to say in such a way that it can't be misunderstood, in a clear and concise way is really not that easy.